### ISSUE TITLE
Syntax error when running phpcov.phar

### ISSUE THREAD

**Issue by lenrsmith** [sebastianbergmann/phpcov#110]
Just downloaded the phar per the Readme and hit the following error on my first attempt at running it:

```
php phpcov.phar
phpcov 8.2.0 by Sebastian Bergmann.
PHP Parse error:  syntax error, unexpected ')' in phar:///Users/leonard/vagrant/waterfluence-vagrant/commercial/phpcov.phar/src/cli/ArgumentsBuilder.php on line 245

Parse error: syntax error, unexpected ')' in phar:///Users/leonard/vagrant/waterfluence-vagrant/commercial/phpcov.phar/src/cli/ArgumentsBuilder.php on line 245
```

Looks like an extra comma in the argument list is the issue. See https://github.com/sebastianbergmann/phpcov/blob/master/src/cli/ArgumentsBuilder.php#L244

**Comment by sebastianbergmann** [comment]
Your PHP is too old.

**Comment by lenrsmith** [comment]
Ok. Thanks for pointing that out. The project I was hoping to to use this with is fixed at 7.2 for now. I'll work out another solution.
